src/uct/ib/rc/base/rc_ep.c

```diff
@@ -99,7 +104,9 @@ ucs_status_t uct_rc_fc_init(uct_rc_fc_t *fc, int16_t winsize
 
 void uct_rc_fc_cleanup(uct_rc_fc_t *fc)
 {
-    UCS_STATS_NODE_FREE(fc->stats);
+    if (UCS_STATS_NODE_CHECK(fc->stats)) {
```

Comment:
maybe UCS_STATS_NODE_FREE would be able to ignore NULL?

Reply:
done